src/game.h

```diff
@@ -13,6 +14,7 @@ struct GameState {
     Asset::System asset_system = {};
     GFX::Renderer renderer = {};
     Input::Input input = {};
+    Audio::AudioStruct *audio_struct;
```

Comment:
Motivation for why you don't save the audio as data? Is it because it doesn't really make sense for audio to be "replayed"?